### DISCUSSION TITLE
Sharing shortcuts

### DISCUSSION THREAD

**Discussion by sebastian-65** [Ulauncher/Ulauncher#1119] [Ideas]
Both me and my girlfriend use ulauncher. And it would be really useful and handy, if two shortcuts.json files could be sourced - one that could be shared via any sharing tool, and then solely mine / hers for only our own stuff that the other person doesn't need to have.

Is there any workaround how to achieve this setup?

**Comment by friday** [comment]
Why not symlink the file?

**Reply by sebastian-65** [reply]
The problem is how to source two different files on the single computer at the same time - one file for shared shortcuts, the other one for personal shortcuts. How this can be achieved? Will symlink help in this case somehow?

**Reply by friday** [reply]
There is only one file. You can use symlinks to share that file between accounts. If that's not enough you have to handle this manually I guess.
